stories/Undo.stories.tsx

```diff
@@ -1,11 +1,36 @@
+import {
+  Meta,
+  Story
+} from '@storybook/react/types-6-0';
 import React, { useState } from 'react';
 import { Canvas } from '../src/Canvas';
-import { Node, Edge, MarkerArrow, Port, Icon, Arrow, Label, Remove, Add, NodeProps, EdgeProps } from '../src/symbols';
-import { UndoRedoEvent, useUndo } from '../src/helpers';
+import {
+  UndoRedoEvent,
+  UndoRedoHistory,
+  useUndo
+} from '../src/helpers';
+import {
+  Add,
+  Arrow,
+  Edge,
+  Icon,
+  Label,
+  MarkerArrow,
+  Node,
+  Port,
+  Remove
+} from '../src/symbols';
 
 export default {
   title: 'Demos/Undo Redo',
   component: Canvas,
+  argTypes: {
+    initialHistory: {
+      control: {
+        disable: true // TODO make it readonly when it'll be possible - See https://github.com/storybookjs/storybook/issues/14048
```

Comment:
move TODO above the line

Reply:
FYI I've been working on this on Storybook https://github.com/storybookjs/storybook/pull/14077